docs/turn-howto.md

```diff
@@ -151,8 +151,8 @@ This will install and start a systemd service called `coturn`.
 
     NB: If your TLS certificate was provided by Let's Encrypt, TLS/DTLS will
     not work with any Matrix client that uses Chromium's WebRTC library. This
-    currently includes Element Android/iOS; see their [respective](https://github.com/vector-im/element-android/issues/1533)
-    [issues](https://github.com/vector-im/element-ios/issues/2712) for more details.
+    currently includes Element Android & iOS. For more details, read the underlying
+    [WebRTC issue](https://bugs.chromium.org/p/webrtc/issues/detail?id=11710).
```

Comment:
Thanks! I think it's useful to keep the element-android and element-ios links too. Sorry, I probably didn't express this well.

```suggestion
    currently includes Element Android & iOS. For more details, see their [respective](https://github.com/vector-im/element-android/issues/1533)
    [issues](https://github.com/vector-im/element-ios/issues/2712)
    and the underlying
    [WebRTC issue](https://bugs.chromium.org/p/webrtc/issues/detail?id=11710).
```

Reply:
(I'm not sure what's going on in that diff there)